app/models/cmr.rb

```diff
@@ -686,6 +688,7 @@ def self.get_cmr_base_url
     if cmr_base_url.nil?
       cmr_base_url = 'https://cmr.earthdata.nasa.gov'
     end
+    cmr_base_url = 'https://cmr.earthdata.nasa.gov'
```

Comment:
this either seems redundant or that you can remove all the lines above it in the method.

Reply:
debugging accidently made it into the commit.  removed.